TestProject/output/ConsoleOutput.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace output;
+
+use options\Getopt;
+use cellularAutomat\Board;
+
+class ConsoleOutput extends BaseOutput
+{
+    function outputBoard(Board $_board)
+    {
+        for ($y = 0; $y < $_board->getHeight(); ++$y) {
+            for ($x = 0; $x < $_board->getWidth(); ++$x) {
+                $_board->setCell($x, $y, rand(0, 1));
```

Comment:
Why are you calling `setCell()´ here? The purpose of this method is to print the board to the console not randomly fill the board!